web/static/js/referenceColors.js

```diff
@@ -0,0 +1,28 @@
+// @flow
+import range from 'lodash/range'
+
+const referenceColors = [
+  '#673ab7', '#3f51b5', '#2196f35', '#03a9f4',
+  '#00bcd4', '#009688', '#4caf50', '#8bc34a',
+  '#cddc39', '#ffeb3b', '#ffc107', '#ff9800',
+  '#ff5722', '#f44336', '#e91e63', '#9c27b0'
+]
+
+const singleColor = '#4faf55'
+
+export function referenceColorClasses(totalNeeded: number) {
+  let nextColorIncrement = referenceColors.length / totalNeeded
+  return range(0, totalNeeded).map((i) => {
+    return 'referenceColor' + (i * nextColorIncrement)
```

Comment:
It would be nice to define what happens if we exceed the maximum number of colors defined. Maybe we can find a way to reuse the same colors but changing the saturation.

At least we could try not to explode and show what we can with as much color differentiation as possible (ie, if more than 16 colors, show the remaining lines black, or reuse the colors). We can safely assume that we won't have more than 16 questionnaires, but for quotas I cannot be so sure.